src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -60,17 +83,27 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
   <div id="container">
 
     <h1>
+      <!-- Conversation title -->
       <%= conversation.getTitle() %>
-      <i onclick="toggleSettingsDropdown()" class="fa fa-cog"> </i>
+        
+      <!-- Setting button and content -->
+      <div id="dropdown-settings">
+        <i onclick="toggleSettingsDropdown()" class="fa fa-cog"> </i>
+        <div class="dropdown-content" style="display:none">
+          <li> <form action="/chat/<%= conversation.getTitle() %>" method="POST">
```

Comment:
Now that we're doing AJAX calls (using the `fetch` API), how about we replace these forms with simple buttons that call the endpoint using `fetch`?